hc/accounts/tests/test_profile.py

```diff
@@ -21,22 +21,30 @@ def test_it_sends_set_password_link(self):
         self.assertTrue(len(token) > 10)
 
         ### Assert that the token is set
+        self.assertTrue(token)
 
         ### Assert that the email was sent and check email content
+        self.assertEqual(len(mail.outbox), 1)
+        self.assertIn("Here's a link to set a password for your account on My Monitoring Project:", mail.outbox[0].body)
+
+
 
     def test_it_creates_api_key(self):
         self.client.login(username="alice@example.org", password="password")
 
         form = {"create_api_key": "1"}
-        r = self.client.post("/accounts/profile/", form)
+        response = self.client.post("/accounts/profile/", form)
+        
         ### Test it creates API key
+        self.assertTrue(response)
```

Comment:
This might pass even when an unexpected response is returned. Please check how it was tested before